db/migrate/20161114214352_add_role_to_list_membership.rb

```diff
@@ -0,0 +1,5 @@
+class AddRoleToListMembership < ActiveRecord::Migration[5.0]
+  def change
+    add_column :list_memberships, :role, :integer, null: false, default: 10
```

Comment:
But yes, it is probably more efficient to store a variety of states as integers and map them to strings in the application